src/pudl/transform/ferc714.py

```diff
@@ -319,6 +340,270 @@ def _pre_process(df: pd.DataFrame, table_name: str) -> pd.DataFrame:
     return out_df
 
 
+def melt_hourx_columns_csv(df):
```

Comment:
a lot of these smaller functions (especially the `_csv` suffixed ones) are just code that I pulled out of the hourly function. I pulled them out for several reasons:
- i wanted to be able to intersperse transform.classes functions with existing ferc714 specific functions. i think it looks much nicer and more readable with the main table function mostly being a chain of various transform functions.
- i needed to add many steps into the transform of this table. if i kept everything in there it would have exploded.
- these smaller steps are all now much much more unit test-able bc they have clear inputs and contained scopes
- now you can tell which pieces should pertain to which stage of the processing with the function suffix (`_csv`, `_xbrl`, `_instant_xbrl`, `_duration_xbrl`, or nothing (the concated table)).

The main downside i see here is that you can't easily see which of these functions are general or specific to a particular table. I think we should transform the two other tables and see if there is some way to clump them reasonably even just with module headers.